### PR TITLE
[bitnami/metrics-server] Add tests and publishing using VIB

### PR DESCRIPTION
**Pull request by FraPazGal**
### Description of the change

The main objective of this PR is to publish our Bitnami Metrics Server Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/FraPazGal/charts/pull/113

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/metrics-server/vib-publish.json [hunk 1]
```diff
@@ -16,6 +16,42 @@
         }
       ]
     },
+    "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/metrics-server"
+        },
+        "runtime_parameters": "c2VydmljZUFjY291bnQ6CiAgY3JlYXRlOiB0cnVlCiAgYXV0b21vdW50U2VydmljZUFjY291bnRUb2tlbjogdHJ1ZQpleHRyYUFyZ3M6Ci0gLS1hdXRob3JpemF0aW9uLWFsd2F5cy1hbGxvdy1wYXRocz0vbWV0cmljcwotIC0ta3ViZWxldC1pbnNlY3VyZS10bHM9dHJ1ZQpjb250YWluZXJQb3J0czoKICBodHRwczogODQ0NApzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cHM6IDQ0Mwpjb250YWluZXJTZWN1cml0eUNvbnRleHQ6CiAgZW5hYmxlZDogdHJ1ZQogIHJ1bkFzVXNlcjogMTAwMg==",
```
Comment (joancafom) [inline .vib/metrics-server/vib-publish.json:25]: You are missing the `rbac.create=true` param here, which I believe is important for the chart to work properly

**Review by joancafom — APPROVED** [review]
Overall, it LGTM. Just added a minor comment.

File: .vib/metrics-server/vib-publish.json (hunk 1 above)
Comment (FraPazGal) [reply .vib/metrics-server/vib-publish.json]: You are right, changed now!

**Review by joancafom — APPROVED** [review]
LGTM!

**Pull request merged**